pegasus/sites.v3/hourofcode.com/public/map.haml

```diff
@@ -13,5 +14,11 @@ layout: wide_index
 
   = view :header
 %h2.center-text= view :hoc_events_counter
```

Comment:
The /map page on our site never switched over to the static map during hoc. This portion uses the static map during hoc or if the user is from Morocco